python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: Copyright (c) 2023-2024, NVIDIA CORPORATION & AFFILIATES.
```

Comment:
This is failing style checks for a really gross reason. This is too long of a line. There's one other case where I could see this, but I don't love the solution:

https://github.com/rapidsai/cudf/blob/af65d52c7d4ca41606482926bdcc001644b7d108/python/cudf/cudf/core/resample.py#L1-L3

Reply:
can the copyright section be exempted from check somehow? It's not really a "comment" in the normal sense I'd say.

Reply:
You could `# noqa` but that’s even noisier and not better imo. I am inclined to say that if we require SPDX compatible license headers, we have to increase the line length limits.

cc: @shwina for input

Reply:
I updated this to 83.